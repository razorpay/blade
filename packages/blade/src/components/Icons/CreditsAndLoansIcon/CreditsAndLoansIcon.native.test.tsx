import CreditsAndLoansIcon from '.';
import renderWithTheme from '~utils/testing/renderWithTheme.native';

describe('<CreditsAndLoansIcon />', () => {
  it('should render CreditsAndLoansIcon', () => {
    const renderTree = renderWithTheme(
      <CreditsAndLoansIcon color="feedback.icon.neutral.intense" size="large" />,
    ).toJSON();
    expect(renderTree).toMatchSnapshot();
  });
});
