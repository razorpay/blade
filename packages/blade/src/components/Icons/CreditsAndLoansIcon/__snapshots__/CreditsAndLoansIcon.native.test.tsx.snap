// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CreditsAndLoansIcon /> should render CreditsAndLoansIcon 1`] = `
<View
  style={
    {
      "flex": 1,
    }
  }
>
  <RNSVGSvgView
    accessibilityElementsHidden={true}
    align="xMidYMid"
    bbHeight="20px"
    bbWidth="20px"
    data-blade-component="icon"
    fill="none"
    focusable={false}
    height="20px"
    importantForAccessibility="no-hide-descendants"
    meetOrSlice={0}
    minX={0}
    minY={0}
    style={
      [
        {
          "backgroundColor": "transparent",
          "borderWidth": 0,
        },
        [
          {},
        ],
        {
          "flex": 0,
          "height": 20,
          "width": 20,
        },
      ]
    }
    vbHeight={24}
    vbWidth={24}
    width="20px"
  >
    <RNSVGGroup
      fill={null}
      propList={
        [
          "fill",
        ]
      }
    >
      <RNSVGPath
        clipRule={0}
        d="M8.51363 6.17762C5.92422 6.82722 4 9.18368 4 12C4 14.8163 5.92422 17.1728 8.51364 17.8224C6.96606 16.3636 6 14.2946 6 12C6 9.70539 6.96605 7.63642 8.51363 6.17762ZM11.9731 4.25901C11.8009 4.21381 11.6264 4.17426 11.4499 4.14057C10.9658 4.0482 10.467 4 9.95782 4C5.55715 4 2 7.58743 2 12C2 16.4126 5.55715 20 9.95782 20C10.6438 20 11.3108 19.9125 11.948 19.7475L11.9443 19.7334C12.6003 19.9073 13.2893 20 14 20C18.4183 20 22 16.4183 22 12C22 7.58172 18.4183 4 14 4C13.2997 4 12.6204 4.08998 11.9731 4.25901ZM14 6C10.6863 6 8 8.68629 8 12C8 15.3137 10.6863 18 14 18C17.3137 18 20 15.3137 20 12C20 8.68629 17.3137 6 14 6Z"
        fill={
          {
            "payload": 4280563015,
            "type": 0,
          }
        }
        fillRule={0}
        propList={
          [
            "fill",
            "fillRule",
          ]
        }
      />
    </RNSVGGroup>
  </RNSVGSvgView>
</View>
`;
